catalyst/dl/callbacks/checkpoint.py

```diff
@@ -96,13 +107,15 @@ def on_epoch_end(self, state: RunnerState):
         if state.stage.startswith("infer"):
             return
 
-        checkpoint = self.pack_checkpoint(
+        valid_metrics = dict(state.metrics.valid_values)
+
+        checkpoint = utils.pack_checkpoint(
             model=state.model,
             criterion=state.criterion,
             optimizer=state.optimizer,
             scheduler=state.scheduler,
             epoch_metrics=dict(state.metrics.epoch_values),
-            valid_metrics=dict(state.metrics.valid_values),
+            valid_metrics=valid_metrics,
```

Comment:
if you change `valid_metrics`... maybe it's better to do the same `for epoch_metrics`?

Reply:
why? i use `valid_metrics` for my calculations, and don't use `epoch_metrics`

Reply:
you don't use `valid_metrics` in this scope

Reply:
updated